torch/_torch_docs.py

```diff
@@ -619,6 +619,105 @@ def merge_dicts(*dicts):
     True
 """)
 
+add_docstr(torch.all,
+           r"""
+all() -> BoolTensor
```

Comment:
alll(input) -> Tensor.

You can mention that it returns a tensor of bools in the description.